packages/circuits/test/draw.test.js

```diff
@@ -35,6 +35,8 @@ describe("Draw Cards Test", () => {
         handRoot = mimcsponge.multiHash([handHash1, handHash2]);
     });
 
+    // set longer timeout for test
+    jest.setTimeout(10000);
```

Comment:
What does this do exactly? (Note I'm not familiar with Jest)

Reply:
Jest by default has a 5000ms timeout, so if the test does not complete within 5s, it's considered as fail. Since some of this proof generation takes a little longer, I set it to a longer period to make sure they can be run to complete.